Makefile

```diff
@@ -1,8 +1,11 @@
 start:
 	@docker-compose up -d
 
+stop:
+	@docker-compose down -v
```

Comment:
I'd expect this to run `docker-compose stop` - but I'd suggest dropping both `stop` and `start` alltogether, because they don't really add any value.

Reply:
True, removed them